pages/view/_iscnId/index.vue

```diff
@@ -99,6 +99,14 @@
         </template>
       </Button>
       <Button
+        v-if="isNFTBook"
+        preset="secondary"
+        class="w-full lg:w-auto"
+        :text="$t('NFTPortal.button.mint.book')"
+        @click="clickMintNFTBook"
+      />
```

Comment:
Should modify `isShowMintButton` to check for NFT book mint status, now it only check for WNFT API
Maybe also check if current wallet is owner of ISCN for `isShowMintButton`